### PR TITLE
Major updates

### PR DESCRIPTION
**Pull request by fsimkovic**
## Added
- ``conkit.plot`` subpackage refactored to allow ``matplotlib`` access of ``Figure`` instances. This provides
  functionality similar to ``seaborn``, so ``matplotlib.Axes`` can be provided into which a plot is drawn.
- ``ContactMap.as_list`` function to represent the contact map as a 2D-list of residue indexes
- ``conkit.misc.normalize`` function to apply Feature scaling normalization
- ``CONTRIB.rst`` file to list all contributors
- ``SequenceFile.diversity`` property defined by :math:`\sqrt{N}/L`
- ``ContactMap.reindex`` to reindex a contact map given a new starting index
- ``ContactMap.singletons`` returns a copy of the contact map with singleton contacts, i.e. ones without neighbors
## Changed
- Changed API interface for ``conkit.plot`` in accordance to necessary changes for above
- ``ContactMapFigure`` now accepts ``lim`` parameters for axes limits
- ``ContactMapFigure`` and ``ContacctMapChordFigure`` improved to better space marker size
- Typos corrected in documentation

### CODE REVIEW

**Comment by hlasimpk** [comment]
Note to self: I'm up to e0c3616

no obvious problems so far!

File: conkit/core/sequencefile.py [hunk 1]
```diff
@@ -131,6 +131,21 @@ def is_alignment(self):
                 break
         return self._status == SequenceAlignmentState.aligned
 
+    @property
+    def diversity(self):
+        """The diversity of an alignment defined by :math:`\sqrt{N}/L`. 
+        
+        ``N`` equals the number of sequences in
+        thethe alignment and ``L`` the sequence length
```
Comment (hlasimpk) [inline conkit/core/sequencefile.py]: Typo 'thethe'

File: conkit/core/contactmap.py [hunk 2]
```diff
@@ -322,16 +322,18 @@ def singletons(self):
            Cannot find SciPy package
```
Comment (hlasimpk) [inline conkit/core/contactmap.py]: This error will no longer exist as the code no longer uses scipy

File: conkit/core/mappings.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    R = 15
+    S = 16
+    T = 17
+    U = 21
+    V = 18
+    W = 19
+    X = 21
+    Y = 20
+    Z = 21
+
```
Comment (hlasimpk) [inline conkit/core/mappings.py]: I presume there is a logic to this, but why are B, J, O, U, X and Z all set to 21?

Comment (hlasimpk) [reply conkit/core/mappings.py]: Oh, just remembered what the letters were representing and how many amino acids there are

File: conkit/core/sequencefile.py (hunk 1 above)
Comment (fsimkovic) [reply conkit/core/sequencefile.py]: Will be added in a later PR.

**Pull request merged**